plugin/trino-base-jdbc/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
 
     <artifactId>trino-base-jdbc</artifactId>
-    <description>Trino - Base JDBC Connector</description>
+    <description>Trino - Base JDBC connector</description>
```

Comment:
```suggestion
    <description>Trino - Base JDBC connector support</description>
```

or library .. since this is NOT actually a connector

Reply:
connector library

Reply:
ok